bindata/hypershift/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -0,0 +1,866 @@
+---
+kind: StatefulSet
```

Comment:
This needs an AntiAffinity to ensure we run all 3 replicas in different AZs (and nodes).

Reply:
Added pod anti affinity.